packages/base/src/3dview/helpers.ts

```diff
@@ -67,7 +67,6 @@ export interface IMouseDrag {
 
 export interface IMeshGroupMetadata {
   type: string;
```

Comment:
Something I should have added in my PR:

```typescript
  jcObject: IJcadObject;
```

Reply:
thanks, it'll be there with my next commit.